src/relay/pass/alpha_eq.cc

```diff
@@ -246,6 +264,36 @@ struct AlphaEq : ExprFunctor<void(const Expr&, const Expr&)> {
       equal = false;
     }
   }
+
+  void VisitExpr_(const IfNode *op, const Expr& e2) final {
```

Comment:
add additional testcase to test_pass_alpha_eq, which causes previous test to fail